File: backend/src/ILSpy.Host/Model/MemberData.cs
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT license. See the LICENSE file in the project root for more information.

using ICSharpCode.Decompiler.TypeSystem;

namespace ILSpy.Host
{
    public class MemberData
    {
        public string Name { get; set; }

        public int Token { get; set; }

        public TypeKind MemberSubKind { get; set; }
    }
}

File: vscode-extension/src/protocol/MemberData.ts
/*------------------------------------------------------------------------------------------------
 *  Copyright (c) 2021 ICSharpCode
 *  Licensed under the MIT License. See LICENSE.TXT in the project root for license information.
 *-----------------------------------------------------------------------------------------------*/

import { MemberSubKind } from "../decompiler/MemberSubKind";

export default interface MemberData {
  name: string;
  token: number;
  subKind: MemberSubKind;
}
